tests/scripts/expect/_common.exp

```diff
@@ -69,25 +70,31 @@ proc spawn_node {id {type ""} {radio_url ""}} {
 
     send_user "\n# ${id} ${type}\n"
 
+    if {[info exists ::env(CC)] && $::env(CC) == "clang"} {
```

Comment:
It seems `llvm-cov` supports `GCOV_PREFIX` too. <https://llvm.org/docs/CommandGuide/llvm-cov.html>

Any chance to make this also applied to LLVM?

Reply:
But there's no `gcov-tool merge` for `llvm-cov`

Reply:
OK, I see. Look good.